src/functionalTest/java/uk/gov/hmcts/reform/wataskmonitor/MonitorTaskJobControllerForInitiationJobTest.java

```diff
@@ -62,9 +64,92 @@ public void task_initiation_job_should_initiate_task_and_taskState_should_be_una
             .statusCode(HttpStatus.OK.value())
             .body(is(expectedResponse.apply(JobName.INITIATION.name())));
 
-        Map<String, CamundaVariable> camundaVariableMap =
+        Map<String, CamundaVariable> camundaVariables =
             common.getTaskVariables(authenticationHeaders, taskVariables.getTaskId());
-        assertEquals("unassigned", camundaVariableMap.get("taskState").getValue());
+
+        assertEquals(taskVariables.getCaseId(), camundaVariables.get("caseId").getValue());
+        assertEquals("unassigned", camundaVariables.get("taskState").getValue());
+
+    }
+
+    @Test
+    public void task_initiation_job_should_not_initiate_delayed_task_and_taskState_should_be_unconfigured() {
+
+        TestVariables taskVariables = common.setupDelayedTaskAndRetrieveIds();
+
+        assertNotNull(taskVariables);
+        assertNotNull(taskVariables.getCaseId());
+        assertNotNull(taskVariables.getTaskId());
+
+        caseIds.add(taskVariables.getCaseId());
+
+        given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, serviceToken)
+            .body(TestUtility.asJsonString(new MonitorTaskJobRequest(new JobDetails(JobName.INITIATION))))
+            .when()
+            .post("/monitor/tasks/jobs")
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .body(is(expectedResponse.apply(JobName.INITIATION.name())));
+
+        Map<String, CamundaVariable> camundaVariables =
+            common.getTaskVariablesFromCamunda(authenticationHeaders, taskVariables.getTaskId());
+
+        String caseId = ((HashMap) (((HashMap) camundaVariables).get("caseId"))).get("value").toString();
+        String actualTaskState = ((HashMap) (((HashMap) camundaVariables).get("taskState"))).get("value").toString();
+        Object actualCftTaskState = ((HashMap) camundaVariables).get("cftTaskState");
+
+        assertEquals(taskVariables.getCaseId(), caseId);
+        assertEquals("unconfigured", actualTaskState);
+        assertNull(actualCftTaskState);
```

Comment:
Same here I don't think this is correct:
since you are casting/converting the CamundaVariable into a Object.
and then attempting to assert using:
```
assertEquals("unconfigured", actualTaskState);
```

this should be 
```
assertEquals("unconfigured", actualTaskState.getValue());
```

Since the CamundaVariable is an object that contains a value and a type see: https://github.com/hmcts/wa-task-monitor/blob/master/src/main/java/uk/gov/hmcts/reform/wataskmonitor/domain/camunda/CamundaVariable.java